src/gl/gl3device.cpp

```diff
@@ -30,6 +31,8 @@ bool32 needToReadBackTextures;
 int32   alphaFunc;
 float32 alphaRef;
 
+bool isAdreno = false;
```

Comment:
Perhaps create a quirks struct:
```
struct {
    bool reset_depth_before_viewport;
} rw_gl_quirks;
```

and use `rw_gl_quirks.reset_depth_before_viewport` elsewhere?